crates/nu-command/src/commands/into_int.rs

```diff
@@ -1,76 +1,173 @@
 use crate::prelude::*;
 use nu_engine::WholeStreamCommand;
 use nu_errors::ShellError;
-use nu_protocol::{Primitive, ReturnSuccess, Signature, SyntaxShape, UntaggedValue, Value};
+use nu_protocol::{
+    ColumnPath, Primitive, ReturnSuccess, Signature, SyntaxShape, UntaggedValue, Value,
+};
 
-use num_bigint::ToBigInt;
+use num_bigint::{BigInt, ToBigInt};
 
 pub struct IntoInt;
 
 #[derive(Deserialize)]
 pub struct IntoIntArgs {
-    pub rest: Vec<Value>,
+    pub rest: Vec<ColumnPath>,
 }
 
+#[async_trait]
```

Comment:
I cannot compile locally and am unsure why:
```
error: cannot find attribute `async_trait` in this scope
  --> crates/nu-command/src/commands/into_int.rs:17:3
   |
17 | #[async_trait]
   |   ^^^^^^^^^^^

```
Does this need to be async? 
I did not see other uses of `#[async_trait]` so would prefer to remove this.

Reply:
I think Jonathan changed it yesterday where we don't use async any longer.